dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jsonbImpl-2.0.0.feature

```diff
@@ -0,0 +1,14 @@
+# This private impl feature corresponds to JSON-B 2.0 with the Yasson implementation
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jsonbImpl-2.0.0
+singleton=true
+visibility=private
+-features=com.ibm.websphere.appserver.jsonp-2.0, \
+  com.ibm.websphere.appserver.classloading-1.0, \
+  com.ibm.websphere.appserver.jakarta.cdi-3.0,\
+  com.ibm.websphere.appserver.javaeeCompatible-8.0
```

Comment:
perhaps now would be the time to create a `javaeeCompatible-9.0` feature?

Reply:
Yes.  Remember from previous meetings that the feature will be named eeCompatible-9.0.

Reply:
Yes.  Remember from previous meetings that the feature will be named eeCompatible-9.0.  The feature is added.  All jakarta features in the working branch to date that declared a dependency on javaeeCompatible-8.0 were changed to eeCompatible-9.0.